tests/test-cases/Group1-Docker-Commands/1-02-Docker-Pull.robot

```diff
@@ -89,7 +89,7 @@ Pull the same image concurrently
     \   Log  ${res.stdout}
     \   Log  ${res.stderr}
     \   Should Be Equal As Integers  ${res.rc}  0
-    \   Should Contain  ${res.stdout}  Downloaded newer image for library/redis:latest
+    \   Should Contain  ${res.stdout}  Downloaded newer image for default-project/redis:latest
```

Comment:
This is remote registry specific - we can use a RegExp match to avoid hardcoding details about the remote project name we're using.

Reply:
Agree but I would prefer a separate PR for any improvements. This is "as is" update.